### PR TITLE
Feat/deploy merkle distributor

### LINKED ISSUE #214: Add merkle distributor deployment scripts
> [Notion Project Page](https://www.notion.so/mentolabs/MiniPay-Airdrop-Claiming-2d226ec13270425692ce4096dd302184?pvs=4)

### Overview

After #517, include the contract as a submodule to the `mento-deployments` repo and create two deployment scripts:
- cUSD Merkle Distributor 
- Mento Merkle Distributor
Each will get a merkle root and the respective token addresses.

### Acceptance criteria

-  [ ] Script is created.
-  [ ] Contracts are deployed and tested on testnets with a dummy merkle tree.

### PR DESCRIPTION
**Pull request by philbow61**
### Description

This Pr adds the deployment script for deploying two MerkleDistributor contracts for the Minipay airdrop.
One for the Mento Allocation and one for the cUSD Allocation.
The deployment is done through the create command since the contract is written in 0.8.17 and our utils are incompatible. The deadline is set to 91 days after the deployment of this contract based of c.a 30.4 days per month => 3 months. 
The current roots in the root.json files are mock values used to verify the deployment correctly picks them up.

### Tested

ran the script on Alfajores and verified the contract is configured correctly.

### Related issues

- Fixes #214

### CODE REVIEW

File: script/upgrades/MINIDROP/deploy/MINIDROP-00-Create-MerkleDistributor.sol [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+contract MINIDROP_CreateMerkleDistributor is Script {
+  using Contracts for Contracts.Cache;
+
+  function run() public {
+    contracts.load("MUGOV-00-Create-Factory", "latest");
+    IRegistry registry = IRegistry(0x000000000000000000000000000000000000ce10);
+    address cUSD = registry.getAddressForStringOrDie("StableToken");
+    address MENTO = IGovernanceFactory(contracts.deployed("GovernanceFactory")).mentoToken();
+
+    // Transfer ownership to mentoLabsMultiig for drain mechanism
```
Comment (bayological) [inline script/upgrades/MINIDROP/deploy/MINIDROP-00-Create-MerkleDistributor.sol:26]: nit: slightly misleading comment

File: script/upgrades/MINIDROP/deploy/MINIDROP-00-Create-MerkleDistributor.sol [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    address mentoDistributor;
+
+    vm.startBroadcast(vm.envUint("MENTO_DEPLOYER_PK"));
+    {
+      cUSDDistributor = deployMerkleDistributor(cUSD, merkleRootCUSD, endTime);
+      console.log("MerkleDistributor for cUSD deployed at:", cUSDDistributor);
+      mentoDistributor = deployMerkleDistributor(MENTO, merkleRootMENTO, endTime);
+      console.log("MerkleDistributor for MENTO deployed at:", mentoDistributor);
+
+      IOwnableLite(cUSDDistributor).transferOwnership(mentoLabsMultisig);
```
Comment (bayological) [inline script/upgrades/MINIDROP/deploy/MINIDROP-00-Create-MerkleDistributor.sol:45]: No need to change but you could also do this in one line without creating the interface:

(bool success,) = cUSDDistributor.call(abi.encodeWithSignature("transferOwnership(address)",mentoLabsMultisig));

**Review by bayological — APPROVED** [review]
LGTM. 

Consider adding some error handling, especially when doing the reading from the file.

**Review by chapati23 — COMMENTED** [review]
lgtm

**Review by chapati23 — APPROVED** [review]

**Pull request merged**